src/checkstatus/get_dbstatus_function.py

```diff
@@ -20,7 +23,7 @@ def lambda_get_dbinstance_status(event, context):
 
 def eval_dbinstance_status(rds_client, context, taskname, identifier):
     max_attempts = util.get_waiter_max_attempts(context)
-    if taskname == constants.SNAPSHOT:
+    if taskname == constants.SNAPSHOT or taskname == constants.COPY_SNAPSHOT:
```

Comment:
ok changing it as more elegant, although lists imho are required if there is more than 2 elements. Here seemed a bit unnecessary as you explicitly see what the condition is. But anyway, changing it.

Reply:
@despot this doesn't seem unnecessary to me. I think what @namitad  is suggesting here is a construct as follows.
`if taskname in [constants.SNAPSHOT, constants.COPY_SNAPSHOT]`
which is good since it allows for future proofing the conditional and reducing if else ladders.

Reply:
@stationeros yes, I already did this as agreed.